config/s2i/gold/jenkins-persistent-buildconfig-template.yaml

```diff
@@ -0,0 +1,275 @@
+          volumeMounts:
+          - name: "${JENKINS_SERVICE_NAME}-data"
+            mountPath: "/var/lib/jenkins"
+          terminationMessagePath: "/dev/termination-log"
+          imagePullPolicy: IfNotPresent
+          capabilities: {}
+          securityContext:
+            capabilities: {}
+            privileged: false
+        volumes:
+        - name: "${JENKINS_SERVICE_NAME}-data"
+          persistentVolumeClaim:
+            claimName: "${JENKINS_SERVICE_NAME}"
+        restartPolicy: Always
+        dnsPolicy: ClusterFirst
+- kind: ServiceAccount
+  apiVersion: v1
+  metadata:
+    name: "${JENKINS_SERVICE_NAME}"
+    annotations:
+      serviceaccounts.openshift.io/oauth-redirectreference.jenkins: '{"kind":"OAuthRedirectReference","apiVersion":"v1","reference":{"kind":"Route","name":"${JENKINS_SERVICE_NAME}"}}'
+- kind: RoleBinding
+  apiVersion: v1
+  metadata:
+    name: "${JENKINS_SERVICE_NAME}_edit"
+  groupNames:
+  subjects:
+  - kind: ServiceAccount
+    name: "${JENKINS_SERVICE_NAME}"
+  roleRef:
+    name: edit
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: "${JNLP_SERVICE_NAME}"
+  spec:
+    ports:
+    - name: agent
+      protocol: TCP
+      port: 50000
+      targetPort: 50000
+      nodePort: 0
+    selector:
+      name: "${JENKINS_SERVICE_NAME}"
+    type: ClusterIP
+    sessionAffinity: None
+- kind: Service
+  apiVersion: v1
+  metadata:
+    name: "${JENKINS_SERVICE_NAME}"
+    annotations:
+      service.alpha.openshift.io/dependencies: '[{"name": "${JNLP_SERVICE_NAME}",
+        "namespace": "", "kind": "Service"}]'
+      service.openshift.io/infrastructure: 'true'
+    creationTimestamp:
+  spec:
+    ports:
+    - name: web
+      protocol: TCP
+      port: 80
+      targetPort: 8080
+      nodePort: 0
+    selector:
+      name: "${JENKINS_SERVICE_NAME}"
+    type: ClusterIP
+    sessionAffinity: None
+parameters:
+- description: Git source URI for Jenkins S2I
+  name: REPO_URL
+  required: true
+  value: https://github.com/CentOS-PaaS-SIG/ci-pipeline.git
+- description: Git branch/tag reference
+  name: REPO_REF
+  value: master
+- description: Path within Git project to build; empty for root project directory.
+  name: CONTEXT_DIR
+  value: config/s2i/jenkins/master
+- name: JENKINS_SERVICE_NAME
+  displayName: Jenkins Service Name
+  description: The name of the OpenShift Service exposed for the Jenkins container.
+  value: jenkins
+- name: JNLP_SERVICE_NAME
+  displayName: Jenkins JNLP Service Name
+  description: The name of the service used for master/slave communication.
+  value: jenkins-jnlp
+- name: ENABLE_OAUTH
+  displayName: Enable OAuth in Jenkins
+  description: Whether to enable OAuth OpenShift integration. If false, the static
+    account 'admin' will be initialized with the password 'password'.
+  value: 'true'
+- name: MEMORY_LIMIT
+  displayName: Memory Limit
+  description: Maximum amount of memory the container can use.
+  value: 512Mi
```

Comment:
@p3ck thoughts? I know I always up this. If this is 'gold,' shouldn't it be a value we don't all have to up? To the best of my knowledge, you up it too.

Reply:
Yes - We have found that 512MB is way too low.  We should move this to 2GB